exercises/node-javascript/add-esdk-complete/retrieve.js

```diff
@@ -0,0 +1,25 @@
+/* ADD-ESDK-START:
```

Comment:
@mattsb42-aws @lizroth Here is the complete retrieve.  The esdk has been added...